corehq/apps/api/odata/serializers.py

```diff
@@ -140,6 +140,35 @@ def to_json(self, data, options=None):
             config_id
         )
         data.pop('meta')
-        data['value'] = data.pop('objects')
+
+        config = CaseExportInstance.get(config_id)
+        data['value'] = self.serialize_cases_using_config(data.pop('objects'), config)
 
         return json.dumps(data, cls=DjangoJSONEncoder, sort_keys=True)
+
+    @staticmethod
+    def serialize_cases_using_config(cases, config):
+        selected_columns = [
+            column for column in config.tables[0].columns
+            if column.selected
+        ]
+        return [
+            {
+                column.label: _get_case_value_by_column(case_data, column)
+                for column in selected_columns
+            }
+            for case_data in cases
+        ]
+
+
+def _get_case_value_by_column(case_data, column):
+    lookup_key = _get_lookup_key_from_column(column)
+    return case_data.get(lookup_key, None) or case_data['properties'].get(lookup_key, None)
```

Comment:
I would have thought that this needs to do the same thing exports does so it can also transpose things like `owner_name` or `parent.name`.

Reply:
Updated: 249713d0fc056221d678d1d5b8bedc8c37a45675

Reply:
nice! relieved it was straightforward.